lib/react_client/react_interop.dart

```diff
@@ -32,6 +32,15 @@ abstract class ReactDom {
   static Element findDOMNode(object) => ReactDOM.findDOMNode(object);
   static ReactComponent render(ReactElement component, Element element) => ReactDOM.render(component, element);
   static bool unmountComponentAtNode(Element element) => ReactDOM.unmountComponentAtNode(element);
+
+  /// Returns a a portal that renders [children] into a [container].
+  ///
+  /// Portals provide a first-class way to render children into a DOM node that exists outside the DOM hierarchy of the parent component.
+  ///
+  /// [children] can be any renderable React child, such as a [ReactElement], [String], or fragment.
+  ///
+  /// See: <https://reactjs.org/docs/portals.html>
+  static ReactPortal createPortal(dynamic children, Element container) => ReactDOM.createPortal(children, container);
```

Comment:
@greglittlefield-wf @joebingham-wk @kealjones-wk just confirming that this is an intentional addition in a major release instead of waiting for the subsequent minor?

Reply:
That's right, so that consumers (like WSD) are able to transition from `react_dom.render` to portals.